tools/accuracy_checker/accuracy_checker/adapters/mask_rcnn_with_text.py

```diff
@@ -0,0 +1,153 @@
+"""
+Copyright (c) 2019 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import cv2
+import numpy as np
+
+from .mask_rcnn import MaskRCNNAdapter
+from ..config import StringField, ConfigError, NumberField
+from ..representation import TextDetectionPrediction
+from ..utils import contains_all
+
+
+class MaskRCNNWithTextAdapter(MaskRCNNAdapter):
+    __provider__ = 'mask_rcnn_with_text'
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'classes_out': StringField(
+                description="Name of output layer with information about classes. "
+                            "(optional, if your model has detection_output layer as output).",
+                optional=True
+            ),
+            'scores_out': StringField(
+                description="Name of output layer with bbox scores."
+                            "(optional, if your model has detection_output layer as output).",
+                optional=True
+            ),
+            'boxes_out': StringField(
+                description="Name of output layer with bboxes."
+                            "(optional, if your model has detection_output layer as output).",
+                optional=True
```

Comment:
you can made outpus required (`optional=False`), it eliminates the need to check they in `configure()`